apps/events/models.py

```diff
@@ -344,14 +344,22 @@ def repetitions(self) -> str:
         return self.params["repetitions"]
 
     def as_string(self, as_timezone: str | None = None):
```

Comment:
Shouldn't we include the schedule name in this text?

Reply:
Whoops yes. I'll update

Reply:
https://github.com/dimagi/open-chat-studio/pull/597/commits/cffe3a0c369f2563059bebfc536ced5bf03f687e.